io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties

```diff
@@ -405,7 +405,7 @@ App.TimeOfUseTariff.Tibber.accessToken.label = Token
 App.TimeOfUseTariff.Tibber.accessToken.description = To link to your Tibber account you need a personal access token. You can create this under "developer.tibber.com/settings/access-token".
 App.TimeOfUseTariff.Tibber.filterForHome.label = Filter for Home
 App.TimeOfUseTariff.Tibber.filterForHome.description = For multiple 'Homes', add either an ID (format UUID) or 'appNickname' for unambiguous identification
-App.TimeOfUseTariff.Tibber.multipleHomesCheck.label = Do you have more than one contract connected to�your Tibber account?
+App.TimeOfUseTariff.Tibber.multipleHomesCheck.label = Do you have more than one contract connected to�your Tibber account?
```

Comment:
Looks like vscodium incorrectly introduced this change

Reply:
Message to merge dev: I just blew up my branch trying to correct this but haven't committed anything yet. Is it possible for you to cherry pick all the rest of the edits but drop the changes to `io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties` please ?

Reply:
:pray:

Reply:
No problem. Thanks for the contribution again!

You can always reset a file to `develop` via `git checkout develop io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties`